engine/security-api/src/test/scala/pl/touk/nussknacker/ui/security/api/PermissionSyntaxTest.scala

```diff
@@ -21,14 +20,26 @@ class PermissionSyntaxTest extends FunSuite with Matchers {
       u.can(p) shouldEqual c
     }
   }
+
   test("Admin permission grands other permissions") {
-    PermissionSyntax.containsPermissionOrAdmin(Permission.Read)(Permission.Admin) shouldBe true
+    def admin(cp: Map[String, Set[Permission]]) = LoggedUser("admin", cp, Option.apply(true))
```

Comment:
Option(true), and later Option(false)?

Reply:
Now is just LoggedUser("admin", cp, true).